### ISSUE TITLE
bug: autorun on windows builds

### ISSUE THREAD

**Issue by YashKandalkar** [efroemling/ballistica#218]
## Description
This is an old issue with the pc builds of bombsquad and I couldn't find an issue addressing it so thought of opening an issue.
I confirmed this with many other pc players, they have the same issue too. So the character start running when pressing just the movement keys without any other keys being pressed. This happens anytime during the game, no specific way to reproduce this; but is happens mid-game and it's annoying that the character just runs and cannot walk. It gets fixed if we rejoin the server or by opening the settings window. 

## Steps to reproduce
1. Launch BombSquad
2. Go to 'gather'
3. Join a server
4. Witness autorun after playing for some time.

## Expected behavior
Character shouldn't run without pressing an action or any other keys on pc

## Machine
**Platform**: Windows 10 
**BombSquad version**: [1.5.28](https://github.com/efroemling/ballistica/releases/tag/v1.5.28)

## Screenshots
None

## Extra
Do not know why it happens, but we have to open the settings window again and again to fix it.

**Comment by ritiek** [comment]
Do you accidentally hold down on some key just before you open the chat window in-game?

I have a guess on what might be happening. For example if you hold down the key for moving in the left direction (using <kbd>A</kbd> or <kbd>←</kbd>) and then open the chat window while the key is held. Now if you release the movement key and close the chat window, the character will still keep moving in the left direction. This seems to happen because the game does not register in-game controls that is pressing and releasing of keys while the chat window is open, so the release event of the movement key is never received properly by the game.

Now replace this movement key with any other key (since running can happen with holding down on any key), and which seems to be happening in your case.

**Comment by YashKandalkar** [comment]
@ritiek I do not think this is happening because the auto run just starts anytime *mid-game*. It's not triggered by opening the chat window. 

And I use h-punch, i-grab, k-bomb, j-run, spacebar-jump. So don't think it has anything to do what keys I press. 

Many other pc players have this big too.

**Comment by efroemling** [comment]
If you can find a way to consistently reproduce this it would be very helpful in fixing it. (such as the aforementioned idea of holding down a key while opening a menu). Perhaps a combination of keys being held down and released in specific orders, etc.

**Comment by YashKandalkar** [comment]
I could not find a specific way to reproduce this, but I asked more PC players and they do not have this bug. Some of them do but most of them don't, maybe there's just something wrong with my keyboard then, it maybe registering a key which I am not pressing. You can close this issue if you want.

Also thankyou for creating this wonderful game, and still working on making it better 👍

**Comment by Dliwk** [comment]
Hmm, I have this problem too, but I thought it was due to the frequent change of keyboard layout. But then again, this is a rather strange behavior that I didn't manage to trigger intentionally.

By the way, I observed this in a local game when I was the host.

**Comment by Loup-Garou911XD** [comment]
Closing as outdated
